app/views/layouts/application.html.erb

```diff
@@ -5,7 +5,7 @@
     <title><%= content_for?(:title) ? yield(:title) : "DARIA" %></title>
     <meta name="description" content="<%= content_for?(:description) ? yield(:description) : "DCAF" %>">
     <meta name='ROBOTS' content='NOINDEX, NOFOLLOW'>
-    <%= javascript_pack_tag 'application' %>
+    <%= javascript_pack_tag 'application', 'data-search-pseudo-elements': true %>
```

Comment:
this is necessary because we're doing some `before` selector stuff; fontawesome can handle this but it requires some extra junk. I don't think it quite rises to the level of tech debt to figure out how to do the grid stuff without this but it was kind of a pain to figure out.

Reply:
can you take a before and after screenshot? i'm trying to figure out how the before selector is used.

Reply:
added a screener!